### ISSUE TITLE
Prevent full-screen mode losing its focus when exiting a game

### ISSUE THREAD

**Issue by Starbuck1991** [JosefNemec/Playnite#3795]
### Feature description

I use a Windows HTPC frontend called Flex to launch several game launchers, such as Steam and Playnite. However, while Steam’s Big Picture mode works flawlessly (gaining focus, executing a game, exiting the game, regaining focus, launching another game, etc.), Playnite loses focus when closing a game. As a result, it remains running but hidden beneath the frontend, and it cannot be controlled until pressing Alt+Tab. Unfortunately, this behavior negates the usefulness of it's full-screen mode operated by a game controller.

**Comment by JosefNemec** [comment]
We already refocus Playnite after game exits, if it doesn't work properly, then some other app is likely preventing Playnite from getting focus properly. You can read more about it [here](https://api.playnite.link/docs/manual/gettingStarted/helpAndTroubleshooting/troubleshooting.html#controller-input-doesnt-work-in-fullscreen-mode-after-exiting-a-game).

**Comment by kcajjones** [comment]
I have this exact issue when running Playnite fullscreen and launching Retroarch v1.9.1 (64bit) then closing Retroarch and Playnite returns the window to fullscreen from minimized but does not regain focus. I can't see how Retroarch could be interfering with it once closed?

**Comment by Starbuck1991** [comment]
> I can't see how Retroarch could be interfering with it once closed?

Exactly. Besides, no other frontend (Steam, GOG, you name it) shows this behavior under the same conditions. Ergo, something must be happening with Playnite and only with Playnite.
